pages/dashboard/dashboard.go

```diff
@@ -11,40 +12,47 @@ import (
 const maxPosts = 5
 const maxFollowing = 5
 
-// Get dashboard.
 func Get(ctx *aero.Context) string {
 	user := utils.GetUser(ctx)
 
 	if user == nil {
 		return frontpage.Get(ctx)
 	}
 
-	posts, err := arn.AllPostsSlice()
-
-	if err != nil {
-		return ctx.Error(500, "Error fetching posts", err)
-	}
+	return Dashboard(ctx)
+}
 
-	arn.SortPostsLatestFirst(posts)
+// Get dashboard.
+func Dashboard(ctx *aero.Context) string {
+	var user *arn.User
+	var posts []*arn.Post
+	var err error
 
-	if len(posts) > maxPosts {
-		posts = posts[:maxPosts]
-	}
+	flow.Parallel(func() {
+		user = utils.GetUser(ctx)
```

Comment:
This is a single Get request to the database, performing in less than a millisecond.
GetUser can be outside of the flow Parallel block.
Instead, fetching all the users you are following should be running in that block.